src/main/java/frc/robot/subsystems/intake/IntakeIOSim.java

```diff
@@ -5,27 +5,31 @@
 import edu.wpi.first.units.MutableMeasure;
 import edu.wpi.first.units.Units;
 import edu.wpi.first.units.Velocity;
+import edu.wpi.first.wpilibj.Timer;
 import lib.motors.SparkMaxSim;
 import lib.motors.TalonFXSim;
 
 public class IntakeIOSim implements IntakeIO {
     private final TalonFXSim angleMotor;
     private final SparkMaxSim frontRoller;
     private final SparkMaxSim centerRoller;
-    private final PositionVoltage positionRequest = new PositionVoltage(0);
+    private PositionVoltage positionRequest = new PositionVoltage(0);
 
     public IntakeIOSim() {
         angleMotor =
                 new TalonFXSim(
-                        1, IntakeConstants.GEAR_RATIO, 0.3, 360 * IntakeConstants.GEAR_RATIO);
+                        1, IntakeConstants.GEAR_RATIO, 0.003, 360 * IntakeConstants.GEAR_RATIO);
         frontRoller = new SparkMaxSim(1, 1, 0.5, 1);
         centerRoller = new SparkMaxSim(1, 1, 0.5, 1);
         angleMotor.setController(IntakeConstants.angleController);
     }
 
     @Override
     public void setAngle(MutableMeasure<Angle> angle) {
+        inputs.angleSetPoint = angle;
+        inputs.currentAngle = Units.Degrees.of(75).mutableCopy();
         angleMotor.setControl(positionRequest.withPosition(angle.in(Units.Degrees)));
+        System.out.println(inputs.currentAngle);
```

Comment:
There is no need for this. I know it's tempting to just spew print statements for debugging, but you don't need that -- all inputs are sent over NetworkTables, and you can watch them using AdvantageScope. This method is faster (no need to redeploy after adding each print) and will also yield better results -- AdvantageScope provides richer data then what you see printed in the console.

Reply:
yeah I know, it was temporary.